starlette/middleware/wsgi.py

```
@@ -0,0 +1,140 @@
+import asyncio
+import io
+import sys
+import typing
+from concurrent.futures import ThreadPoolExecutor
+from starlette.types import ASGIApp, ASGIInstance, Message, Receive, Scope, Send
```

Ruff (0.11.9)

`starlette.types.ASGIApp` imported but unused

Remove unused import

(F401)